docs/ig/use/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Jiri
+meta:
+  - property: og:title
+    content: Jiri | Ethereum
+lang: ig
+---
+
+# Eji Ethereum {#using-ethereum}
+
+<div class="featured">
+  **I chọrọ ịmalite i di na eji Ethereum?** Peeji a ga enye gi ozi di mkpa i choro. Otu. [Jiri ngwa komputa aruru na Ethereum] (#_1-Jiri-ngwa komputa-aruru-na-Ethereum) Abuo. [Gini bu ETH, kedu ka m ga-esi nweta ya?] (#_2-what-is-eth-and-how-do-i-get-it) Ato. [Gịnị bụ obere akpa ego, ma kedu nke m kwesịrị eji?](#_3-what-is-a-wallet-and-which-one-should-i-use)
```

Comment:
These anchor links are incorrect. See /use page of other language translations for reference.